Lib/modulefinder.py

```diff
@@ -619,8 +619,9 @@ def replace_paths_in_code(self, co):
             if isinstance(consts[i], type(co)):
                 consts[i] = self.replace_paths_in_code(consts[i])
 
-        return types.CodeType(co.co_argcount, co.co_kwonlyargcount,
-                              co.co_nlocals, co.co_stacksize, co.co_flags,
+        return types.CodeType(co.co_argcount, co.co_posonlyargcount,
```

Comment:
This construction is not future-proof, it breaks each time we add an argument to the Code constructor. Maybe we need a .replace() method to replace a few fields and copies other, similar to datetime.datetime.replace() (datetime.datime is also immutable).

That's a note unrelated to your PR ;-)